Common/Data/Custom/TradingEconomics/TradingEconomicsCalendar.cs

```diff
@@ -32,6 +39,15 @@ public class TradingEconomicsCalendar : BaseData
     {
         // Filtered calendar event name
         private string _filteredEvent;
+        private string _originalEvent;
+        private const string _baseUrl = "https://api.tradingeconomics.com";
+        private static bool _useStreaming = Config.GetBool("trading-economics-use-streaming");
+        private static string _apiKey = Config.Get("trading-economics-api-key");
+
+        /// <summary>
+        /// Delimiter used to separate country from ticker in <see cref="TradingEconomics.Calendar"/> entries
+        /// </summary>
+        public const string Delimiter = "//";
```

Comment:
Is there a reason for choosing this delimiter?

Reply:
We needed a character that would not clash with other `Symbol` implementations and also characters that are not invalid (`|` and ` `).